src/signTx.h

```diff
@@ -159,6 +159,9 @@ typedef struct {
 		ins_sign_tx_witness_context_t witnesses_ctx;
 	} txPartCtx;
 
+	bool poolOwnerByPath;
```

Comment:
Do we need this variable? I think tx signing mode fully determines whether `poolOwnerPath` is meaningful or not. (it makes sense to add an assert into the security policy to validate the argument it receives)